cmd/scheduler/app/server.go

```diff
@@ -56,10 +55,6 @@ import (
 
 // Run the volcano scheduler.
 func Run(opt *options.ServerOption) error {
-	if opt.PrintVersion {
-		version.PrintVersionAndExit()
```

Comment:
I have a question: Why do we need to move it to main.go? Can we just return here?

Reply:
Maybe it's in this location running error, but i only need know version info.
https://github.com/volcano-sh/volcano/blob/fb068bde7a0cf80392b225517dd3dd522524091a/cmd/scheduler/main.go#L65-L67

Reply:
Do you have any idea? @googs1025